Improve charts accessibility

MikhailWahib opened the pull request:
## 📖 Description

Added accessibility module provided by HighCharts to improve the charts accessibility.

This will add by default:
       - Improvements for the keyboard navigation
       - ARIA attributes for the charts

for more information visit: https://www.highcharts.com/accessibility

This also will resolve a warning that was appearing on the console:

![Screenshot 2024-06-10 074919](https://github.com/egytech-fyi/egytech-fyi/assets/112858669/206a3170-e2d2-4db5-8f12-ac6d239409ff)


## 🔄 Change Type
_Please delete options that are not relevant._
- [x] Bug fix (non-breaking change which fixes an issue)

## 🧪 How Has This Been Tested?
_Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration._

Manually, using keyboard navigation, and the browser Devtools to ensure that the ARIA attributes have been applied.

## 🚧 Affected Parts:
_Details on which features/pages of the app are affected._

All Highcharts charts.

## 📋 Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works

iahmedgamal commented:
Thanks, @MikhailWahib, for your contribution 
can you provide this section? 

![image](https://github.com/egytech-fyi/egytech-fyi/assets/16765061/cde37d66-9735-404b-8252-e34f15f375a1)

Khalifa1997 commented:
Thanks alot for your contribution @MikhailWahib 👏

MikhailWahib commented:
> Thanks, @MikhailWahib, for your contribution can you provide this section?
> 
> ![image](https://private-user-images.githubusercontent.com/16765061/338284347-cde37d66-9735-404b-8252-e34f15f375a1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgwNjQ4OTIsIm5iZiI6MTcxODA2NDU5MiwicGF0aCI6Ii8xNjc2NTA2MS8zMzgyODQzNDctY2RlMzdkNjYtOTczNS00MDRiLTgyNTItZTM0ZjE1ZjM3NWExLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjExVDAwMDk1MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ3MDZkYWQ0MDdmZDdhY2EzMjhjMWI4YTczYjY0MDg0MTA3Y2UyMTllN2NiYTM4ZjJiOTg3ZTVhMWMxYWU2YTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.0rGqO4rEouIcIhlNVpnIrlHkhfxi_MlbFJq0SyPjhPo)

Hi @Khalifa1997, I have updated my PR. Also, I have updated the description for more information.
